evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -187,21 +186,6 @@ def _create_pipelines(self):
                 self.problem_type, self.allowed_model_families
             )
             allowed_estimators = self._filter_estimators(allowed_estimators)
-            if (
```

Comment:
Now that the datetime information is not being set as an index for ARIMA, the time interval of `MS - Month Start` is no longer a concern.